scripts/clean-old-packages

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env python3
+"""
+Clean up old packages, specifying how many to keep
+
+Example:
+    ./clean-old-packages securedrop-yum-test/workstation/buster-nightlies 7
+
+This script is run in CI in a Fedora container. You can spin up a similar
+container locally using podman or docker, e.g.:
+
+    podman run -it --rm -v $(pwd):/workspace:Z fedora:39 bash
```

Comment:
Probably the trickiest part was to get the script running locally on a non-Fedora system, so I added this to help with bootstrapping.